### DISCUSSION TITLE
cssOrigin Precedence

### DISCUSSION THREAD

**Discussion by erosman** [mdn/mdn-community#616] [Content]
Can someone verify that the MDN information is as it should be (and/or maybe explain it to me)?

**Note:** Identical rules are compared in different `cssOrigin` e.g.
```
"author" vs "user"
p { color: red; } vs p { color: blue; }
p { color: red !important; } vs p { color: blue !important; }
```
> [tabs.insertCSS()](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/tabs/insertCSS)
> - "user" enables you to prevent websites from overriding the CSS you insert: see Cascading order.
> - "author" stylesheets behave as if they appear after all author rules specified by the web page.

From above, it is understood that "user" has a higher priority and can override "author".
Therefore the expectation is:
- normal user rule > normal author rule
- important user rule > important author rule

Looking at the table on [Cascading order](https://developer.mozilla.org/en-US/docs/Web/CSS/Cascade#cascading_order) it seems that is not the case.
In tests, the same normal rule inserted in "user" does NOT override the same rule inserted in "author".

**Comment by pepelsbey** [comment]
Closing as outdated/irrelevant.
